src/register/views/RegisterPage.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-main id="register-page">
+    <v-container id="form-container">
+      <v-row class="register-row" justify="center">
+        <v-col cols="12" sm="12" md="8" lg="8">
+          <h1>Register</h1>
+          <register-form @user-infor="register"></register-form>
```

Comment:
The name of the event should be a verb. 
```suggestion
          <RegisterForm @on-submit="onSubmit" />
```